### PR TITLE
Improve relationships between entities

### PR DESCRIPTION
**Pull request by hungnguyen94**
This improves the relationships between the entities. All entities are tied to a project.   
Cue still has a bi-directional relation with Script, but maybe we can just keep it that way. They belong together anyway. 

Suggestions are welcome.

### CODE REVIEW

**Comment by rubenwiersma** [comment]
😱 decreased coverage 🙀

File: src/main/java/nl/tudelft/thefirstorder/domain/CameraAction.java [hunk 1]
```diff
@@ -29,9 +29,6 @@
     @Column(name = "name")
     private String name;
 
-    @Column(name = "duration")
-    private Integer duration;
-
```
Comment (rubenwiersma) [inline src/main/java/nl/tudelft/thefirstorder/domain/CameraAction.java]: Why'd you remove duration?

Comment (hungnguyen94) [inline src/main/java/nl/tudelft/thefirstorder/domain/CameraAction.java]: Duration is in the TimePoint entity now.

**Comment by hungnguyen94** [comment]
I will fix the decreased coverage.

File: src/main/resources/config/liquibase/master.xml [hunk 2]
```diff
@@ -5,16 +5,20 @@
     xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
 
     <include file="classpath:config/liquibase/changelog/00000000000000_initial_schema.xml" relativeToChangelogFile="false"/>
-    <include file="classpath:config/liquibase/changelog/20160430231201_added_entity_CameraAction.xml" relativeToChangelogFile="false"/>
+    <!--<include file="classpath:config/liquibase/changelog/20160430231201_added_entity_CameraAction.xml" relativeToChangelogFile="false"/>-->
```
Comment (rubenwiersma) [inline src/main/resources/config/liquibase/master.xml]: Commented could be removed

File: .jhipster/Project.json [hunk 3]
```diff
@@ -15,6 +15,18 @@
             "otherEntityField": "id",
             "ownerSide": true,
             "otherEntityRelationshipName": "project"
+        },
+        {
+            "relationshipName": "player",
+            "otherEntityName": "player",
+            "relationshipType": "one-to-many",
```
Comment (rubenwiersma) [inline .jhipster/Project.json]: Should the player not be connected to the map?

File: .jhipster/Project.json [hunk 4]
```diff
@@ -15,6 +15,18 @@
...
+            "otherEntityRelationshipName": "project"
+        },
+        {
+            "relationshipName": "camera",
+            "otherEntityName": "camera",
+            "relationshipType": "one-to-many",
+            "otherEntityRelationshipName": "project"
```
Comment (rubenwiersma) [inline .jhipster/Project.json]: Same for camera should it be connected to the map?

**Comment by cadw** [comment]
I agree, improving the realtionships will definitely make the job of implementing the timeline a lot easier. Only the coverage seems to be decreased, so look at that and after that I think we can merge this.

**Comment by hungnguyen94** [comment]
Boom, increased coverage.

**Comment by rubenwiersma** [comment]
Coverage increased! Will merge.

**Pull request merged**